bitrise.yml

```diff
@@ -0,0 +1,184 @@
+workflows:
+  godeps-update:
+      title: Godeps update
+      steps:
+      - script:
+          title: Dependency update
+          inputs:
+          - content: |
+              #!/bin/bash
+              set -ex
+              go get -u -v github.com/tools/godep
+              rm -rf ./Godeps
+              rm -rf ./vendor
+              go get -t -d ./...
+              godep save ./...
+
+  test:  
+    before_run:
+    - _change_workdir
+    - _reset_tmp_dir
+    steps:
+    - script:
+        title: Create empty folder for test_1
+        inputs:
+        - content: mkdir "./folder_to_zip_test_1"
+    - path::./:
+        title: TESTING ZIP folder_to_zip_test_1 dir
+        inputs:
+        - source_path: ./folder_to_zip_test_1
+    after_run:
+        - _test_2_folder_in_folder
+
+  _test_2_folder_in_folder:
+    steps:
+    steps:
+    - script:
+        title: Create folder with empty folder inside for test_2
+        inputs:
+        - content: mkdir -p "./folder_to_zip_test_2/nested_empy_folder_test_2"
+    - path::./:
+        title: TESTING ZIP folder_to_zip_test_2 dir
+        inputs:
+        - source_path: ./folder_to_zip_test_2
+    after_run:
+        - _test_3_file_in_folder
+
+  _test_3_file_in_folder:
+    steps:
+    - script:
+        title: Create folder with text file for test_3
+        inputs:
+        - content: |-
+            mkdir "./folder_to_zip_test_3/" && 
+            touch "./folder_to_zip_test_3/nested_text_test_3.txt"
+    - path::./:
+        title: TESTING ZIP folder_to_zip_test_3 dir
+        inputs:
+        - source_path: ./folder_to_zip_test_3
+    after_run:
+        - _test_4_symlink
+
+  _test_4_symlink:
+    steps:
+    - script:
+        title: Create folder with a symlink to a txt file for test_4
+        inputs:
+        - content: |-
+            touch "symlink_file_test_4.txt" &&
+            mkdir "./folder_to_zip_test_4/" && 
+            cd "./folder_to_zip_test_4" && 
+            ln -s  "../symlink_file_test_4.txt" && 
+            touch "nested_text_test_4.txt"
+    - path::./:
+        title: TESTING ZIP folder_to_zip_test_4 dir
+        inputs:
+        - source_path: ./folder_to_zip_test_4
+    after_run:
+        - _test_5_nested_symlink
+
+  _test_5_nested_symlink:
+    steps:
+    - script:
+        title: Create folder with a nested symlink to a txt file for test_5
+        inputs:
+        - content: |-
+            mkdir "symlink_folder_test_5" &&
+            touch "symlink_folder_test_5/symlink_file_test_5.txt" &&
+            ln -s "symlink_folder_test_5/symlink_file_test_5.txt" &&
+            mkdir "./folder_to_zip_test_5/" &&
+            cd "./folder_to_zip_test_5" &&
+            ln -s  "../symlink_file_test_5.txt"
+    - path::./:
+        title: TESTING ZIP folder_to_zip_test_5 dir
+        inputs:
+        - source_path: ./folder_to_zip_test_5
+
+   # ----------------------------------------------------------------
+   # --- Common workflows
+
+  _change_workdir:
```

Comment:
do not separate single steps in a new workflow if it is not necessary (you use this wf in only one other wf)